Examples/Python/src/Generators.cpp

```diff
@@ -137,6 +137,24 @@ void addGenerators(Context& ctx) {
           "tStdDev",
           &ActsExamples::GaussianDisplacedVertexPositionGenerator::tStdDev);
 
+  py::class_<
+      ActsExamples::UniformPrimaryVertexPositionGenerator,
+      ActsExamples::EventGenerator::PrimaryVertexPositionGenerator,
+      std::shared_ptr<ActsExamples::UniformPrimaryVertexPositionGenerator>>(
+      mex, "UniformVertexGenerator")
+      .def(py::init<>())
+      .def(py::init([](const Acts::Vector4& min, const Acts::Vector4& max) {
```

Comment:
Did you try this with just specifying the constructor arguments to `py::init<>` instead of an explicit lambda?

Reply:
This works if an explicit constructor is defined. Again, for consistency I'd rather keep the same implementation as for other generators.